central/notifiers/pagerduty/pagerduty.go

```diff
@@ -63,9 +63,6 @@ func newPagerDuty(notifier *storage.Notifier, cryptoCodec cryptocodec.CryptoCode
 	decCreds := conf.GetApiKey()
 	var err error
 	if env.EncNotifierCreds.BooleanSetting() {
-		if notifier.GetNotifierSecret() == "" {
```

Comment:
I think we keep this because without an API key you can't call PagerDuty. That said it's not critical to keep the check because an empty token will just fail on the call to PD.